app/models/changeset.rb

```diff
@@ -84,13 +84,32 @@ def find_comparison
 
   def find_pull_requests
     numbers = commits.map(&:pull_request_number).compact
-    numbers.map { |num| PullRequest.find(repo, num) }.compact
+    numbers.map { |num| PullRequest.find(repo, num) }.
+      compact.
+      concat(find_pull_request_for_branch)
   end
 
   def cache_key
     [self.class, repo, previous_commit, commit].join('-')
   end
 
+  def find_pull_request_for_branch
```

Comment:
the `find_` feels redundant

Reply:
also should be `pull_requests` or not return an array ?

Reply:
The `find_` was chosen to match the rest of the code of this class.
For example the `pull_requests` method is used to call `find_pull_requests` and cache the result:
https://github.com/zendesk/samson/blob/b8cc8aeff1abcc803ac699eccd57833564923087/app/models/changeset.rb#L33-L35

I don't mind changing the method name if that's your preference

I'll change the name to plural to indicate that it returns an array 🥇

Reply:
hmm ok consistency is nice too 🤷‍♂️